### PR TITLE
feat: DO NOT MERGE terms of service and privacy policy

### PR DESCRIPTION
**Pull request by ufe-pr**
## Description

I added 2 new pages to the app, Terms of Service and Privacy Policy.
Corresponding links were also included in the Connect Wallet Page and the User Menu drop-down with appropriate styling.

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue

https://github.com/shapeshift/web/issues/963

## Risk

Possibly https://github.com/shapeshift/web/issues/963#issuecomment-1047050272

## Screenshots

The Connect Wallet Page showing the links to the Privacy Policy and Terms of Service
localhost_3000_connect-wallet_returnUrl=_accounts

The Terms of Service Page
localhost_3000_legal_terms-of-service

The User Menu showing the links to the Privacy Policy and Terms of Service
localhost_3000_legal_terms-of-service (1)

The Privacy Policy Page
localhost_3000_legal_terms-of-service (2)

### CODE REVIEW

**Comment by cjthompson** [comment]
These screenshots show a lot of weird formatting issues especially regarding spaces (either lack of spaces after punctuation or additional spaces inside quotes where there shouldn't be).

**Comment by ufe-pr** [comment]
> These screenshots show a lot of weird formatting issues especially regarding spaces (either lack of spaces after punctuation or additional spaces inside quotes where there shouldn't be).
> 
> image image image


I've fixed the formatting issues and created a new commit.

localhost_3000_legal_terms-of-service (5)

localhost_3000_legal_terms-of-service (4)

File: src/pages/Legal/TermsOfService.tsx [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+        <DarkMode>
+          <Center flexDirection={'column'}>
+            <Text as='h3' translation={'connectWalletPage.shapeshift'} />
+            <Text as='h1' translation={'common.terms'} />
+            <Text as='h3' translation={'Dated: January 2022'} mt={4} />
+          </Center>
+          <RawText as='p'>WELCOME TO THE DECENTRALIZED SHAPESHIFT PLATFORM!</RawText>
+          <RawText as='p'>
+            THESE TERMS CONSTITUTE A LEGALLY BINDING AGREEMENT BETWEEN YOU AND THE SHAPESHIFT
+            DECENTRALIZED AUTONOMOUS ORGANIZATION (WHO WE REFER TO IN HE FIRST PERSON IN THESE
```
Comment (0xdef1cafe) [inline src/pages/Legal/TermsOfService.tsx:18]:
```suggestion
            DECENTRALIZED AUTONOMOUS ORGANIZATION (WHO WE REFER TO IN THE FIRST PERSON IN THESE
```

File: src/pages/Legal/TermsOfService.tsx [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+            through our Platform are fully public, they are also fully immutable, meaning that once
+            a transaction has been submitted it is unlikely to be reversible. Additionally, since
+            blockchain transactions are highly technical, there are unfortunately many nefarious
+            actors and scams still prevalent in the space. You should seek independent advice and
+            conduct your own due diligence prior to using our Platform, or quite frankly, doing
+            anything in the cryptocurrency space. Lastly, many cryptocurrencies remain highly
+            volatile. All this is to say that: THE RISK OF LOSS IN BUYING OR SELLING ANY
+            CRYPTOCURRENCY CAN BE SUBSTANTIAL, THEREFORE YOU SHOULD CAREFULLY CONSIDER WHETHER
+            BUYING OR SELLING CRYPTOCURRENCY IS SUITABLE FOR YOU IN LIGHT OF YOUR FINANCIAL
+            CONDITION BEFORE BUYING OR SELLING SUCH CRYPTOCURRENCY. Ok…thanks for bearing with us
```
Comment (0xdef1cafe) [inline src/pages/Legal/TermsOfService.tsx:71]:
```suggestion
            CONDITION BEFORE BUYING OR SELLING SUCH CRYPTOCURRENCY. Ok… thanks for bearing with us
```

File: src/pages/Legal/TermsOfService.tsx [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+              <RawText as='strong'>NFTs</RawText>&quot;) may not be viewable on the Platform's
+              dashboard.
+            </RawText>
+            <RawText as='dd' marginInlineStart={{ base: 2, lg: 4 }}>
+              <RawText as='strong'>
+                Quick note on cryptocurrency pricing information on the Platform:
+              </RawText>{' '}
+              Because the Platform pulls pricing information from external, publicly available
+              sources, we make no guarantees that this pricing information will actually materialize
+              into such value as is listed on the Platform.YOU ARE RESPONSIBLE FOR CONFIRMING AN
```
Comment (0xdef1cafe) [inline src/pages/Legal/TermsOfService.tsx:142]:
```suggestion
              into such value as is listed on the Platform. YOU ARE RESPONSIBLE FOR CONFIRMING AN
```

File: src/pages/Legal/TermsOfService.tsx [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+              annual percentage for a particular smart contract function. The APR for a given smart
+              contract function on our Platform is pulled from that smart contract's publicly
+              available data. Accordingly, you acknowledge that we are not responsible for your
+              reliance on any APR figures that you see on our Platform. For more information on how
+              the underlying APR is calculated, please refer to the website or documentation of the
+              third-party who maintains the applicable smart contract.
+            </RawText>
+          </RawText>
+          <RawText as='p'>
+            <RawText as='strong'>Alpha Platform:</RawText> We have made available an alternate
```
Comment (0xdef1cafe) [inline src/pages/Legal/TermsOfService.tsx:218]: IMO, this entire section regarding `alpha.shapeshift.com` should be removed until it's hosted, and references to the mobile apps should be removed until we host an open source version of the apps, as the current versions are not hosted via the DAO, and we should not blur any lines there.

File: src/pages/Legal/TermsOfService.tsx [hunk 5]
```diff
@@ -0,0 +1,400 @@
...
+            prohibited and open only to individuals who can lawfully participate in such Promotion
+            in that individual's particular jurisdiction.
+          </RawText>
+          <RawText as='p'>
+            <RawText as='strong'>FOX Tokens:</RawText> ShapeShift AG, with whom we are not
+            affiliated, created a standard ERC-20 token on the Ethereum blockchain that can be
+            accessed through most wallets that support Ethereum called the FOX token (each a &quot;
+            <RawText as='strong'>FOX Token</RawText>&quot;). In spring 2021, ShapeShift AG
+            airdropped the majority of its FOX Tokens to prior users, members of various DAOs,
+            shareholders, employees, and to fund us as a DAO. FOX Tokens used as governance tokens
```
Comment (0xdef1cafe) [inline src/pages/Legal/TermsOfService.tsx:313]:
```suggestion
            shareholders, employees, and to fund us as a DAO. FOX Tokens are used as governance tokens
```

File: src/pages/Legal/TermsOfService.tsx [hunk 6]
```diff
@@ -0,0 +1,400 @@
...
+            for the Platform. Neither we nor ShapeShift AG have ever sold FOX Tokens. To learn more
+            about how FOX Token holders can participate in governance of the Platform, please join
+            or discord server{' '}
+            <Link href='https://discord.com/invite/shapeshift'>
+              <RawText as='strong'>here</RawText>
```
Comment (0xdef1cafe) [inline src/pages/Legal/TermsOfService.tsx:318]: add space after here

File: src/pages/Legal/TermsOfService.tsx [hunk 7]
```diff
@@ -0,0 +1,400 @@
...
+            and see the channel titled &quot;governance&quot;. The smart contract for the FOX Tokens
+            can be found{' '}
+            <Link href='https://etherscan.io/token/0xc770eefad204b5180df6a14ee197d99d808ee52d'>
+              <RawText as='strong'>here</RawText>
+            </Link>
+            .
+          </RawText>
+          <RawText as='h2'>LEGAL STUFF</RawText>
+          <RawText as='p'>
+            <RawText as='strong'>Disclaimer of Warranties:</RawText> As we pointed out,the Platform
```
Comment (0xdef1cafe) [inline src/pages/Legal/TermsOfService.tsx:329]:
```suggestion
            <RawText as='strong'>Disclaimer of Warranties:</RawText> As we pointed out, the Platform
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
various formatting changes, and a request to remove a section that is not yet relevant, that IMO should be included at a later date, but IANAL

File: src/pages/Legal/PrivacyPolicy.tsx [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+            carefully. All other aspects of your interaction with the Platform are governed by our{' '}
+            <RawText as='strong'>Terms of Service</RawText>.
+          </RawText>
+          <RawText as='p'>
+            <RawText as='strong'>How we can change this notice:</RawText> We may, in our sole
+            discretion, modify this notice, so you should review this page periodically. When we
+            change this notice, we will update the date at the top of this page. Prior versions of
+            this notice can be found{' '}
+            <Link href='#0'>
+              <RawText as='strong'>here</RawText>
```
Comment (0xdef1cafe) [inline src/pages/Legal/PrivacyPolicy.tsx:30]: add full stop after here

File: src/pages/Legal/PrivacyPolicy.tsx [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+            3Defi, LLC and Zendesk, Inc. (&quot;<RawText as='strong'>Zendesk</RawText>&quot;) to
+            assist in our responses to queries made by users of the Platform and provide higher
+            quality customer support to our users, which is necessary to perform agreement
+            obligations (viz. our Terms of Service) for our users. Zendesk uses cookies as well as
+            the inquiring user's name and email address in order to provide these services. No other
+            information will be shared with Zendesk unless specifically provided by the user.
+            Further information on data protection and your options in connection with Zendesk's
+            services can be found{' '}
+            <Link href='https://www.zendesk.com/company/customers-partners/privacy-policy/'>
+              <RawText as='strong'>here</RawText>
```
Comment (0xdef1cafe) [inline src/pages/Legal/PrivacyPolicy.tsx:90]: add space after here

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
more formatting changes

**Comment by ufe-pr** [comment]
@0xdef1cafe I'm about to implement the suggested changes. Should I also remove the sections concerning the alpha platform and mobile apps?

**Comment by 0xdef1cafe** [comment]
hey @ufe-pr - thanks for following up. after speaking with the lawyer that drafted this, please remove all references to

* mobile apps
* alpha.shapeshift.com
* pendo

we'll add these sections when they're ready and relevant

**Comment by 0xdef1cafe** [comment]
hey @ufe-pr - are you able to action the requested changes please?

**Comment by ufe-pr** [comment]
hi @0xdef1cafe
I've implemented the changes. Please review.

**Comment by 0xdef1cafe** [comment]
hey @ufe-pr thanks for the PR - we're going to consider this done, fix the merge conflicts in https://github.com/shapeshift/web/pull/1093 and get it merged

**Pull request closed without merging**